packages/framework/tests/Unit/PostAuthorTest.php

```diff
@@ -326,6 +326,36 @@ public function testGetMethodNormalizesUsernamesForRetrieval()
         );
     }
 
+    public function testAuthorUsernamesAreNormalizedInTheConfig()
+    {
+        Config::set('hyde.authors', [
+            'foo_bar' => Author::create(),
+            'foo-bar' => Author::create(),
+            'foo bar' => Author::create(),
+            'Foo Bar' => Author::create(),
+            'FOO BAR' => Author::create(),
+        ]);
+
+        $this->assertCount(1, PostAuthor::all());
+    }
+
+    public function testOnlyLastAuthorWithNormalizedUsernameIsKept()
+    {
+        // Technically, a first come, first-served principle would make most sense,
+        // but this is the lightest code implementation and can be used for now.
```

Comment:
Given the context of your application and the goal of preventing unintended duplicates, it makes the most sense to keep the first user in the array matching the normalized username. Here's why:

First-come, first-served principle: This approach respects the order in which authors were added to the configuration. The first occurrence of a normalized username is treated as the "canonical" version.
Predictability: It's easier for users to understand and predict the behavior if they know that the first occurrence will be used.
Backward compatibility: If users have existing configurations, keeping the first occurrence ensures that the system continues to use the author data they originally intended.
Performance: Checking only for the first occurrence can be slightly more efficient than scanning the entire array to find the last occurrence.

Reply:
Implemented in a03541585e646144cf45f47b82ee2f170e6f9f9a